lib/ansible/modules/crypto/get_certificate.py

```diff
@@ -151,7 +151,12 @@ def main():
             module.fail_json(msg="ca_certs file does not exist")
 
     try:
-        cert = get_server_certificate((host, port), ca_certs=ca_certs)
+        int_port = int(port)
+    except Exception as e:
+        module.fail_json(msg="Unable to convert passed port into integer: 0{}".format(e))
```

Comment:
Do you mean `{0}` instead of `0{}`?

Reply:
Yea, I'm getting that fixed but the tests are also complaining about the connection timeout.